src/docs/development/packages-and-plugins/plugin-api-migration.md

```diff
@@ -113,7 +94,7 @@ public class EmbeddingV1Activity extends FlutterActivity {
 }
 ```
 
-7. Add the `EmbeddingV1Activity` to the
+7. (Optional) Add the `EmbeddingV1Activity` to the
```

Comment:
For these optional steps should we mention when they would want to be done?

Reply:
added a few more words